app/controllers/v2/link_sets_controller.rb

```diff
@@ -12,6 +12,14 @@ def put_links
       render status: response.code, json: response
     end
 
+    def get_linked
+      render json: Queries::GetLinked.new(
+        content_id: content_id,
+        link_type: link_type,
```

Comment:
This `link_type` isn't defined anywhere so the endpoint crashes. We should add integration tests that cover the controller.